### PR TITLE
PCHR-2715: Swap Reports Order

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR, the order of Reports is changed in SSP.

## Before
civihr_custom_report___hr17-lih

## After
civihr_custom_report___hr17-lih

## Technical Details
1. Added `crm_custom-select` class to the Pivot table `select`.
---

- [x] Tests Pass

### CODE REVIEW

**Comment by igorpavlov** [comment]
Looks like smth wrong with the markup:

image

`<div>` is closed with `<? php>`

**Comment by deb1990** [comment]
@igorpavlov As per you suggestion, I applied the indentation commit first.

File: civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php [hunk 1]
```diff
@@ -27,110 +27,113 @@ class="panel-body-wrap panel-body-wrap--collapse"
...
-                  pivotTableContainer: jQuery('#reportPivotTable'),
-                  derivedAttributes: initialDerivedAttributes,
-                  tableUrl: '<?php print $tableUrl; ?>',
-                  jsonUrl: '<?php print $jsonUrl; ?>',
-                  filters: <?php print !empty($filters) ? 1 : 0; ?>
-          });
-                  Drupal.behaviors.civihr_employee_portal_reports.instance.show();
-          });
-</script>
+  <script type="text/javascript">
```
Comment (AkA84) [inline civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php]: the indentation is all wrong here

File: civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php [hunk 2]
```diff
@@ -27,110 +27,120 @@ class="panel-body-wrap panel-body-wrap--collapse"
   </div>
```
Comment (AkA84) [inline civihr_employee_portal/templates/civihr-employee-portal-civihr-report-custom.tpl.php]: Seems you have an extra `<div>` before you close the `if`, which seems to be closing the root element, please check

```html
<div id="civihrReports">
  <?php if (!empty($filters)): ?>
    <div ng-controller="FiltersController as filters">
    </div>
  </div>
<?php endif; ?>
```

**Pull request merged**